tests/utils.py

```diff
@@ -313,7 +313,7 @@ def setUpClass(cls):
             os.path.join('dist', 'python-java-support.jar'),
         ])
         cls.jvm = subprocess.Popen(
-            ["java", "-classpath", classpath, "python.testdaemon.TestDaemon"],
+            ["java", "-noverify", "-classpath", classpath, "python.testdaemon.TestDaemon"],
```

Comment:
Yes, it _was_ needed for my local tests on Java 8 – but the need seems to have gone away now that the AST bit is no longer fiddling with `IF` opcode jump targets. Fixed in 0777522.